editor-components/content/content13/content13.module.scss

```diff
@@ -1,96 +1,377 @@
 .container {
+    height: 100%;
     width: 100%;
-    padding: 50px 0;
+    background-image: url("https://craftohtml.themezaa.com/images/vertical-center-line-bg.svg");
```

Comment:
this should be a prop in JSX oart. also we use google storage url for image links